app/models/manageiq/providers/kubevirt/infra_manager.rb

```diff
@@ -150,7 +168,17 @@ def has_credentials?(_type = nil)
   # @param opts [Hash] Additional options to control how to perform the verification.
   #
   def verify_credentials(_type = nil, opts = {})
-    with_provider_connection(opts, &:valid?)
+    with_provider_connection(opts) do |kubevirt|
+      # First check that we're able to connect to the k8s API
+      kubevirt.get_api_group_list
+      # Then check that we can access the /apis/kubevirt.io API group
+      kubevirt.get_api_group_kubevirt_io
```

Comment:
Since we're doing this in 2 spots, I wonder if we should move this into a method.

Reply:
Actually is this a duiplicate method in the same file?